devel/uboot-tools/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                uboot-tools
+github.setup        u-boot u-boot 2020.07 v
+revision            0
+github.tarball_from archive
+license             GPL-2
+categories          devel
+platforms           darwin
+maintainers         {@rubendibattista gmail.com:rubendibattista} openmaintainer
+description         Binary tools for U-Boot.
+long_description    U-Boot is a boot loader for Embedded boards based on \
+                    PowerPC, ARM, MIPS and several other processors. This \
+                    port compiles some binary tools that can be used to \
+                    deploy u-boot on embedded devices.
+
+checksums           rmd160  124de3a5355915ccb2291b4a01f21e7710290521 \
+                    sha256  616b446e15d1cd1ab6461ebb61ac6655a2b13e902fe0601f36c4affb3949d416 \
+                    size    19630685
+
+depends_build       port:gmake
+
+depends_lib         path:include/openssl/bn.h:openssl \
+                    path:include/openssl/rsa.h:openssl \
+                    path:include/openssl/pem.h:openssl \
+                    path:include/openssl/err.h:openssl \
+                    path:include/openssl/evp.h:openssl
```

Comment:
Other ports use `path:lib/libssl.dylib:openssl`, so let's use that here too, ok?

Reply:
By which I mean: just that line, not the others. Only need to depend on openssl once.